common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -920,3 +920,26 @@ def test_video_with_multiple_transcripts_index_dictionary(self):
             },
             "content_type": "Video"
         })
+
+    def test_video_with_no_transcripts_index_dictionary(self):
```

Comment:
Rename this test method since it's not testing the `index_dictionary` method call (as opposed the test methods right above).